src/analysis/retail/monk/mistweaver/modules/spells/EssenceFont.tsx

```diff
@@ -88,22 +138,53 @@ class EssenceFont extends Analyzer {
     this.totalOverhealing += event.overheal || 0;
   }
 
+  chijitooltip() {
+    if(this.chijiActive){
+      return <>Chi-Ji:
+        <ul>
+          <li>
+            {this.chijiGomEFHits} <small>additional Chi-Ji hits</small>
+          </li>
+          <li>
+            {formatNumber(this.chijiGomHealing)} <small>addtional Chi-Ji healing</small>
```

Comment:
Our other tooltips don't use small, so let's not use small here to be consistent. Also, the grey on black might be hard for some people to read.
```suggestion
            {this.chijiGomEFHits} additional Chi-Ji hits
          </li>
          <li>
            {formatNumber(this.chijiGomHealing)} addtional Chi-Ji healing
```